packages/stream/src/index.ts

```diff
@@ -201,6 +201,50 @@ export const createAmplitudeFromStream = (
   return [amplitude, teardown];
 };
 
+/**
+ * Creates a reactive wrapper to get display media streams from screen
+ * ```typescript
+ * [stream, { mutate, refetch, mute, stop } = createScreen(screenSource);
+ * ```
+ * @param screenSource DisplayMediaStreamConstraints | undefined | Accessor<DisplayMediaStreamConstraints | undefined>
+ * @returnValue `stream()` is an accessor to the display media stream (or undefined if not yet loaded)
+ * @property `stream.loading` is a boolean indicator for the loading state
+ * @property `stream.error` contains any error getting the stream encountered
+ * @method `mutate` allows to manually overwrite the stream
+ * @method `refetch` allows to restart the request without changing the constraints
+ * @method `mute` will mute the stream or unmute if called with `false`
+ * @method `stop` allows stopping the display media stream
+ *
+ * The stream will be stopped on cleanup automatically.
+ */
+export const createScreen = (
+  screenSource: MaybeAccessor<DisplayMediaStreamConstraints | undefined>
+): StreamReturn => {
+  const [stream, { mutate, refetch }] = createResource(
+    createMemo<DisplayMediaStreamConstraints | undefined>(() =>
+      typeof screenSource === "function" ? screenSource() : screenSource
+    ),
```

Comment:
If `screenSource` is not a function, there is not need to wrap it in a memo—It'll never change.
```suggestion
    typeof screenSource === "function" ? createMemo(screenSource) : () => screenSource,
```